docs/features/feature-guides/compliance-forms/complete-a-form.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Complete a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Complete a DataHub Compliance Form
```

Comment:
Can remove "DataHub" here.

Reply:
Will leave it in - it helps with SEO to have `DataHub` in H1